src/samVisualize/rendering/configConstants/PhysicsConfig.ts

```diff
@@ -0,0 +1,58 @@
+/*!
```

Comment:
these `FooConfig.ts` types could live in a single file, directly in the `rendering/` folder. Avoid excessive nesting, it makes things harder to find.